django_project/social_users/models.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+import logging
+
+LOG = logging.getLogger(__name__)
+
+import itertools
+from django.contrib.auth.models import User
+from django.contrib.gis.db import models
+
+
+class UserLink(models.Model):
+    """
+    List of user link
+    """
+    link = models.CharField(blank=True, default="", null=True,
+                            max_length=200)
+
+    def __str__(self):
+        return self.link
+
+
+class UserDetail(models.Model):
+    """
+    Extention of User
+    """
+
+    user = models.OneToOneField(User, default=1)
+    link = models.ForeignKey(UserLink, default=None, blank=True, null=True)
+    profile_picture = models.FileField(
```

Comment:
If we are allowing users to upload profile pictures, we should use django hashed storage too - see https://github.com/kartoza/projecta/blob/develop/django_project/core/settings/contrib.py#L69 and we need to have a backup strategy for uploaded media.